docs/cds/cds.rst

```diff
@@ -0,0 +1,452 @@
+              CDS/P/HST/Y                                                   -
+          CDS/P/HST/other                                                   -
+         CDS/P/HST/wideUV                                                   -
+          CDS/P/HST/wideV                                                   -
+     ESAVO/P/HST/ACS-blue                                                   -
+          ESAVO/P/HST/FOC                                                   -
+       ESAVO/P/HST/NICMOS                                                   -
+         ESAVO/P/HST/WFC3                                                   -
+         ESAVO/P/HST/WFPC                                                   -
+        ESAVO/P/HST/WFPC2                                                   -
+
+
+
+Misc
+----
+
+Limiting the number of returned data-sets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another parameter called ``max_rec`` specifies an upper limit for the number of data-sets to be returned:
+
+.. code-block:: python
+
+    >>> cds.query_region(region=cone, max_rec=3)
+         tap_tablename       publisher_id ... vizier_popularity
+    ------------------------ ------------ ... -----------------
+    ivoa.B/assocdata/obscore    ivo://CDS ...               7.0
+          viz7.B/cb/lmxbdata    ivo://CDS ...               2.0
+           vcds1.B/cfht/cfht    ivo://CDS ...               5.0
+
+This astropy table has only 3 rows although we know more data-sets match the query. It's useful if you do not need
+to retrieve all the data-sets matching a query but only a few. Again, the result will come faster from the MOCServer because
+this operation is done at the server side.
+
+Returning a `~mocpy.MOC` object as a result
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some users might want the union of all the MOCs from the data-sets matching the query. You can get a `mocpy.MOC` object
+instead of an `astropy.table.Table` by setting the parameter ``return_moc`` to True. An additional parameter ``max_norder``
+allows the user to set the resolution/precision of the returned MOC that he wants.
+
+As an example, we would like to obtain the union of the spatial coverage of all the Hubble surveys:
+
+.. code-block:: python
+
+    >>> from mocpy import MOC
+    >>> # We want to retrieve all the HST surveys i.e. the HST surveys covering any region of the sky.
+    >>> allsky = CircleSkyRegion(coordinates.SkyCoord(0, 0, unit="deg"), coordinates.Angle(180, unit="deg"))
+    >>> moc = cds.query_region(region=allsky,
+    ... # We want a mocpy object instead of an astropy table
+    ...                        return_moc=True,
+    ... # The order of the MOC
+    ...                         max_norder=7,
+    ... # Expression on the ID meta-data
+    ...                        meta_data="ID=*HST*")
+    >>> moc.plot(title="Union of the spatial coverage of all the Hubble surveys.")
+
+.. image:: ./HST_union.png
```

Comment:
ideally this image could be generated on the fly based on the docs example. However, for that we need to make sure the doctests/build is run with remote data, so I suggest to do this in a follow-up PR, rather than holding up this one.